docs/tutorials/image_classification.md

```diff
@@ -8,11 +8,11 @@ In this tutorial we will learn how to:
 ## Setting up the Project
 
 First, create a new project directory named `image_classification` with the following structure:
-    
```

Comment:
This changes are made by my formatter. I suggest you add a pre-commit hook for this:

```yaml
-   repo: https://github.com/pre-commit/pre-commit-hooks
    rev: v2.3.0
    hooks:
    -   id: end-of-file-fixer
    -   id: trailing-whitespace
```

Reply:
#164